services/migrations/0011_add_gdpr_url.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.10 on 2020-05-11 11:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("services", "0010_add_sensitivedata_permissions")]
```

Comment:
If you rename this file to `0012_add_gdpr_url.py` and change this line to:

`dependencies = [("services", "0011_make_alloweddatafield_field_name_unique")]`

you are able to get rid of the merge migration `services/migrations/0012_merge_20200511_1534.py`